packages/util/src/types/index.ts

```diff
@@ -1,8 +1,6 @@
 import React from 'react'
 
-export type HTMLPropsFor<
-  Tag extends keyof JSX.IntrinsicElements
-> = JSX.IntrinsicElements[Tag]
+export interface HTMLPropsFor<T> extends React.HTMLAttributes<T> {}
```

Comment:
This abstraction has become an alias. Doesn't have to and maybe shouldn't happen this round, but I would like to remove this low-value alias and use the React type directly.

Reply:
Easy to find and replace in this PR. You want to do it?

Reply:
Um, sure, I will do it.

Reply:
Maybe don't I'll take take care of it. It's actually a little more difficult because we use that alias in things other than interfaces. We also use it in type casting a numerous places. I'll do it if you haven't started.

Reply:
Ok, I've stopped.